website/docs/basics/blocks-intro.md

```diff
@@ -0,0 +1,72 @@
+---
+id: blocks-intro
+title: Intro
+sidebar_label: Intro
+---
+
+Before we dive into blocks and see how everything is set, we should describe this setup's mindset.
+If you managed to set up your project by this point with [WP-CLI](wp-cli) command `setup_theme`, you might have a few questions. We will try to answer all those questions here.
+
+### Why all my blocks look the same and have the same name structure?
+
+This is because we defined the name structure for all our blocks and components to be able to load everything automatically. More on this please check the [block structure](block-structure) and [component structure](blocks-component-structure) chapters.
```

Comment:
```suggestion
This is because we defined the name structure for all our blocks and components to be able to load everything automatically. For more info about this please check the [block structure](block-structure) and [component structure](blocks-component-structure) chapters.
```